drivers/adc-dac/ad5592r/ad5592r-base.c

```diff
@@ -231,9 +234,12 @@ int32_t ad5592r_set_channel_modes(struct ad5592r_dev *dev)
 			adc |= BIT(i);
 			break;
 
-		case CH_MODE_GPIO:
-			dev->gpio_map |= BIT(i);
-			dev->gpio_in |= BIT(i); /* Default to input */
+		case CH_MODE_GPI:
+			dev->gpio_in |= BIT(i);
+			break;
+
```

Comment:
nitpick: remove please extra line. no needed

Reply:
Making these corrections now, just to make sure - you would like the end of the CH_MODE_GPI case to be directly above the CH_MODE_GPO case with no space? Like this:
`		case CH_MODE_GPI:
			dev->gpio_in |= BIT(i);
			break;
		case CH_MODE_GPO:
			dev->gpio_out |= BIT(i);
			break;`

Reply:
Yes, indeed. You can use as a reference other drivers that use the switch statement,

Reply:
The original implementation had spaces between each, you're sure you want me to remove them all from this switch/case statement?

Reply:
If the original implementation for all cases has spaces in between, then keep it like that. I did not have a look at the entire driver, but only on your modifications.
Then the spacing is subject to another commit.

Reply:
Perfect, I will leave the spacing as is for the moment